tests/test_sageintacct/test_tasks.py

```
@@ -1838,3 +1841,121 @@ def test_mark_paid_on_fyle(db, mocker):
+
+    mock_search_and_create_vendors = mocker.patch('apps.sage_intacct.utils.SageIntacctConnector.search_and_create_vendors')
+    mocker.patch('apps.sage_intacct.tasks.get_filtered_expense_group_ids', return_value=[expense_group.id])
+    mocker.patch('apps.sage_intacct.tasks.get_employee_as_vendors_name', return_value=['Ashwin T'])
+
+    search_and_upsert_vendors(workspace_id=workspace_id, configuration=configuration, expense_group_filters={}, fund_source='CCC')
+
+    assert mock_search_and_create_vendors.call_count == 1
+
+    configuration.corporate_credit_card_expenses_object = 'JOURNAL_ENTRY'
+    configuration.use_merchant_in_journal_line = False
+    configuration.employee_field_mapping == 'VENDOR'
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)